paper/joss_preview/paper.Rmd

```diff
@@ -0,0 +1,165 @@
+  corresponding: true
+  affiliation: 1
+- name: Joe Thorley
+  orcid: 0000-0002-7683-4592
+  affiliation: 1
+- name: Troy Hegel  
+  affiliation: 3
+- name: Barry Nobert
+  affiliation: 3
+- name: Dave Hervieux
+  affiliation: 3
+affiliations:
+- index: 1
+  name: Poisson Consulting, Canada
+- index: 2
+  name: Integrated Ecological Research, Canada
+- index: 3
+  name: Province of Alberta, Canada
+date: 6 May 2024
+bibliography: paper.bib
+output: 
+  rticles::joss_article: default
+csl: apa.csl
+journal: JOSS
+tags:
+  - boreal caribou
+  - Bayesian
+  - maximum likelihood
+  - survival
+  - recruitment
+  - population growth
+---
+
+# Summary
+
+Although many animal populations are managed based on their abundance [@williams_analysis_2002], reliable abundance estimates can be impractical to obtain for mobile herding populations with large ranges such as boreal caribou (*Rangifer tarandus caribou*).
+Consequently, such populations are commonly managed based on their population growth rate ($\lambda$), which can be estimated from the population recruitment and adult female survival rates [@hatter_moose_1991]. 
+
+The `bbou` suite is a set of R packages [@r_core_team_2023] to facilitate simulation and analysis of boreal caribou survival and recruitment data to estimate population growth.  
+
+It consists of:  
+- [`bboudata`](https://poissonconsulting.github.io/bboudata/), a collection of simulated and anonymized empirical survival and recruitment datasets.  
+- [`bboutools`](https://poissonconsulting.github.io/bboutools/), tools to estimate population growth in a Bayesian or Maximum Likelihood (ML) framework, implemented under the hood using the `Nimble` R package [@de_valpine_programming_2017].  
+- [`bboushiny`](https://poissonconsulting.github.io/bboushiny/), a Graphical User Interface (GUI) to `bboutools` (\autoref{fig:bboushiny}).    
+- [`bbouretro`](https://poissonconsulting.github.io/bbouretro/), tools to estimate population growth using traditional frequentist methods.  
+- [`bbousims`](https://poissonconsulting.github.io/bbousims/), tools to simulate survival and recruitment data from hypothetical collaring and composition surveys.  
+
+Each R package has a website with function documentation and a 'Get Started' guide.
+There are also several vignettes, including on `bboutools` [analytical methods](https://poissonconsulting.github.io/bboutools/articles/methods.html) and [prior selection](https://poissonconsulting.github.io/bboutools/articles/priors.html); `bbouretro` [analytical methods](https://poissonconsulting.github.io/bbouretro/articles/retro-methods.html); using [`bbousims` with `bboutools`](https://poissonconsulting.github.io/bbousims/articles/bboutools.html); and comparison of Bayesian and traditional method estimates with [empirical](https://poissonconsulting.github.io/bbousuite/articles/empirical-comparisons.html) and [simulated](https://poissonconsulting.github.io/bbousuite/articles/simulations.html) data. 
```

Comment:
I think we need to add a to "and comparison of Bayesian" to get "and a comparison of Bayesian"

Reply:
done